moveit_commander/bin/moveit_commander_cmdline.py

```diff
@@ -136,30 +155,55 @@ def run_service(group_name):
     print("Running ROS service")
     rospy.spin()
 
+
 def stop_ros(reason):
     rospy.signal_shutdown(reason)
     roscpp_shutdown()
 
+
 def sigint_handler(signal, frame):
     stop_ros("Ctrl+C pressed")
     # this won't actually exit, but trigger an exception to terminate input
     sys.exit(0)
 
-if __name__=='__main__':
+
+if __name__ == "__main__":
 
     signal.signal(signal.SIGINT, sigint_handler)
 
     roscpp_initialize(sys.argv)
 
-    rospy.init_node('move_group_interface_cmdline', anonymous=True, disable_signals=True)
-
-    parser = argparse.ArgumentParser(usage = """%(prog)s [options] [<group_name>]""",
-                                     description = "Command Line Interface to MoveIt")
-    parser.add_argument("-i", "--interactive", action="store_true", dest="interactive", default=True,
-                        help="Run the command processing script in interactive mode (default)")
-    parser.add_argument("-s", "--service", action="store_true", dest="service", default=False,
-                        help="Run the command processing script as a ROS service")
-    parser.add_argument("group_name", type=str, default="", nargs='?', help="Group name to initialize the CLI for.")
+    rospy.init_node(
+        "move_group_interface_cmdline", anonymous=True, disable_signals=True
+    )
+
+    parser = argparse.ArgumentParser(
+        usage="""%(prog)s [options] [<group_name>]""",
+        description="Command Line Interface to MoveIt",
+    )
+    parser.add_argument(
+        "-i",
+        "--interactive",
+        action="store_true",
+        dest="interactive",
+        default=True,
+        help="Run the command processing script in interactive mode (default)",
```

Comment:
I didn't know ending a list of arguments with an open comma like this wasn't a syntax error. I guess it's just hideous. The more you know.

There's no setting for this in black, right?

Reply:
there is a good reason for this: it avoids syntax errors when (named) arguments get shuffled around -- and the person doing that forgets to add the comma and then spends I-don't-know-how-long figuring out where the syntax error is.

Reply:
I guess. Although when I see a comma followed by a closed bracket I look like this:

<img width="499" alt="Screen Shot 2021-03-23 at 12 40 13" src="https://user-images.githubusercontent.com/4535737/112090752-7f9a9a00-8bd7-11eb-9077-c9441bb32b5d.png">

I also don't know how often people reorder arguments vs how often code is read, but it doesn't matter. It's the setting in black and I don't have strong objections.